impl/client/src/main/javascript/web/pentaho/data/filter/_core/property.js

```diff
@@ -87,7 +88,7 @@ define(function() {
        */
       _contains: function(elem) {
         var prop = this.property;
-        return elem.$type.has(prop) && this._operation(elem.getv(prop));
+        return elem.$type.has(prop) && this._operation(elem.getv(prop), elem.getf(prop));
```

Comment:
Performance reasons would dictate that `_operation` would start receiving the element and would only get what it actually needs.